extension/package.json

```diff
@@ -22,6 +22,7 @@
     "lint": "eslint --ext .ts,.tsx ./src",
     "package": "vsce package --yarn",
     "package:with:engine": "yarn package -o ${npm_package_name}-with-engine-${npm_package_version}.vsix",
```

Comment:
May you have also do adjust this one?

Reply:
Yes, true. But may be you could still change the version inside in the pom? So we see the exact version if we install it this way?

Reply:
changed that, now the output is designer-11.vsix and inside pom the version is 11.3.2023xxxx